src/apiVerificationSteps.ts

```diff
@@ -0,0 +1,112 @@
+import memory from '@qavajs/memory';
+import { Then } from '@cucumber/cucumber';
+import jp from 'jsonpath';
+import { expect } from 'chai';
+
+/**
+ * Comparing response status code with given
+ *
+ * @example
+ * Status Code is equal "200"
+ *
+ * @param {String} statusCode should be valid status code
+ */
+Then('Status Code is {validation} {text}', (validation, statusCode) => {
+  const response = memory.getValue(`${browser.sessionId}response`);
+
+  validation(response.status, parseInt(statusCode, 10));
+});
+
+/**
+ * Verifying that response contains all models
+ *
+ * @example
+ * Response "$.data.items" contains:
+ *       | _id                   |
+ *       | appId                 |
+ *       | serviceCategory       |
+ *
+ * @param {String} pathQuery json path
+ * @param {Array} dataTable given data table with all properties
+ */
+Then('Response {text} contains:', (pathQuery, dataTable) => {
+  const response = memory.getValue(`${browser.sessionId}response`);
+  const property = jp.value(response.body, pathQuery);
+
+  if (Array.isArray(property) && property.length > 0) {
+    property.forEach((data) => {
+      dataTable.rawTable
+        .join()
+        .split(',')
+        .forEach((value: string) => {
+          expect(data).to.have.property(value);
+        });
+    });
+  } else {
+    dataTable.rawTable
+      .join()
+      .split(',')
+      .forEach((value: string) => {
+        expect(property).to.have.property(value);
+      });
+  }
+});
+
+/**
+ * Verifying that response model has necessary type
+ *
+ * @example
+ * Response "$.data.items" is an "array"
+ *
+ * @param {String} pathQuery json path
+ * @param {String} type should be named as expected value type
+ */
+Then('Response {text} is a(n) {text}', (pathQuery, type) => {
+  const response = memory.getValue(`${browser.sessionId}response`);
+  const property = jp.value(response.body, pathQuery);
+
+  expect(property).to.be.an(type);
+});
+
+/**
+ * Verify that array size is equal to|less than|greater than given number
+ *
+ * @example
+ * Response "$.data.items" size is "greater than" "0"
+ *
+ * @param {String} responseElement json path
+ * @param {String} action should be named as expected action (equal to|less than|greater)
+ * @param {String} expectedValue Number for comparing with array size
+ */
+Then('Response {text} size is {text} {text}', (pathQuery, action, expectedValue) => {
+  const response = memory.getValue(`${browser.sessionId}response`);
+  const property = jp.value(response.body, pathQuery);
+  const count = property.length;
+
+  switch (action) {
```

Comment:
validation is already a part of config loader as parameter type {validation}. So definition may be changed to
Response {text} size {validation} {text}